### PR TITLE
Lf/cleanup crew

### PR DESCRIPTION
**Pull request by FrazierLE**
#### What does this PR do?
- [x] Removes unnecessary console logs, extra spacing, and unused imports
- [x] Add some css to login page
- [x] Clear the search bar after searching on addBook page
- [x] Remove commented out code
- [x] Squash login bug
- [x] Squat borrow book bug

#### Where should your reviewer start
`Login`, `BorrowBook`, `AddBook`

#### What (if any) features are you implementing
- Added border-radius to login container
- Added setTimeout to login error and cleared inputs when clicking return to login button
- Cleared search in addBook section

#### What did you refactor
- [x] Removed console logs and unneeded imports

#### Is there anything that you need from your teammate?
Check it out and let me know if I need to change/update anything

#### Any other comments, questions, or concerns?

### CODE REVIEW

File: src/Components/BookDetails/BookDetails.tsx [hunk 1]
```diff
@@ -90,7 +90,7 @@ const BookDetails = () => {
   }
 
   const borrowerOptions = () => {
-    return bookDetails?.users?.map((value: Value, index: number, array: Value[]): JSX.Element => {
+    return bookDetails?.users?.map((user: any): JSX.Element => {
```
Comment (JPierce28) [inline src/Components/BookDetails/BookDetails.tsx:93]: Thank you for this, that should allow all associated tests to pass.

**Review by JPierce28 — APPROVED** [review]
Good work with the clean up and bug fixes.

File: src/Components/BookDetails/BookDetails.tsx (hunk 1 above)
Comment (FrazierLE) [reply src/Components/BookDetails/BookDetails.tsx:93]: Oh perfect!

**Comment by Adelle-Pitsas** [comment]
I pull it down and it looks good to me! thanks fo picking up those last minute bugs!

**Pull request merged**